Daily-Dev-Challenges/day8/url-shortener/frontend/style.css

```
@@ -0,0 +1,57 @@
+/* style.css */
+body {
+  font-family: 'Inter', sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)